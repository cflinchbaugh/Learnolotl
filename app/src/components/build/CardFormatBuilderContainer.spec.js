import React from 'react';
import {shallow} from 'enzyme';
import {CardFormatBuilderContainer} from './CardFormatBuilderContainer';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('CardFormatBuilderContainer', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const tree = renderer.render(
            <CardFormatBuilderContainer />
        );

        expect(tree).toMatchSnapshot();
    });
});  