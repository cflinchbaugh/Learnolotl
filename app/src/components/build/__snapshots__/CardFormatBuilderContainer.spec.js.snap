// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFormatBuilderContainer renders and matches snapshot 1`] = `
<CardFormatBuilder
  dynamicFormElementFactoryData={
    Object {
      "addInputField": [Function],
      "formElementData": undefined,
      "formElements": undefined,
      "handleInputChange": [Function],
    }
  }
  saveFormatButtonData={
    Object {
      "onClickFunction": [Function],
      "type": "next",
    }
  }
/>
`;
